frontend/CLAUDE.md

```
@@ -495,6 +495,13 @@ const routes = {
   - `/api/auth/verify/`
   - `/api/auth/logout/`
 
+### Analytics & Campaign Attribution (`src/lib/analytics.js`)
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)